src/ast.rs

```diff
@@ -832,13 +832,16 @@ fn parse_jsdoc_dynamic_import(input: &str) -> monch::ParseResult<JsDocImport> {
   use monch::*;
   let original_input = input;
```

Comment:
I caused by accident by moving this line before the next one for the end of the function.